web3.js/test/stake-program.test.ts

```diff
@@ -387,8 +387,9 @@ describe('StakeProgram', () => {
       const STAKE_ACCOUNT_MIN_BALANCE =
         await connection.getMinimumBalanceForRentExemption(StakeProgram.space);
 
-      // todo: use `Connection.getMinimumStakeDelegation` when implemented
-      const MIN_STAKE_DELEGATION = LAMPORTS_PER_SOL;
+      const MIN_STAKE_DELEGATION = (
+        await connection.getStakeMinimumDelegation()
+      ).value;
```

Comment:
nit: I know it was like this when you found it, but awaiting these in parallel will speed up the test.

```typescript
const [
  SYSTEM_ACCOUNT_MIN_BALANCE,
  SYSTEM_ACCOUNT_MIN_BALANCE,
  MIN_STAKE_DELEGATION,
] = await Promise.all([
  connection.getMinimumBalanceForRentExemption(0),
  connection.getMinimumBalanceForRentExemption(StakeProgram.space),
  connection.getStakeMinimumDelegation(),
]);
```